HerPublicWebsite.BusinessLogic/ExternalServices/EmailSending/GovUkNotifyApi.cs

```diff
@@ -135,8 +132,34 @@ string followUpLink
             
 
         }
-    }
+        
+        public void SendComplianceEmail(
+                MemoryStream recentReferralRequestOverviewFileData,
+                MemoryStream recentReferralRequestFollowUpFileData,
+                MemoryStream historicReferralRequestFollowUpFileData
+                )
+                {
+                    var template = govUkNotifyConfig.ComplianceReportTemplate;
+                    Dictionary<String, dynamic> personalisation = new Dictionary<String, dynamic>
+                    {
+                        { "File1Link", NotificationClient.PrepareUpload(recentReferralRequestOverviewFileData.ToArray(), true)},
+                        { "File2Link", NotificationClient.PrepareUpload(recentReferralRequestFollowUpFileData.ToArray(), true)},
+                        { "File3Link", NotificationClient.PrepareUpload(historicReferralRequestFollowUpFileData.ToArray(), true)},
+                    };
+                    foreach (var emailAddress in govUkNotifyConfig.ComplianceEmailRecipients)
```

Comment:
(as discussed, I suspect it'll be easier to make this a comma or semicolon separated list)